Translation corrections (#92)
07th-mod/higurashi-console-arcs: Update/some_007.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>山瀬</color>", NULL, "<color=#f5e6d3>Yamase</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_yamase1023", 256, TRUE);
	OutputLine(NULL, "「いいですか！",
			NULL, "\"Listen!", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_yamase1024", 256, TRUE);
	OutputLine(NULL, "　いくらこの部屋に入居しているのが今はあなただけでも、勝手に部屋をこんなふうに汚されては困ります！",

> 			NULL, " You may be the only one interred in this room for now, but that doesn't mean you can just cause a mess like this without permission!", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " You may be the only one occupying this room at the moment but that doesn't mean you can just cause a mess like this without permission!", Line_WaitForInput);
```
